frappe/desk/reportview.py

```diff
@@ -60,9 +60,8 @@ def get_count() -> int | None:
 		return frappe.call(controller.get_count, args=args, **args)
 
 	args.distinct = sbool(args.distinct)
-	distinct = "distinct " if args.distinct else ""
 	args.limit = cint(args.limit)
-	fieldname = f"{distinct}`tab{args.doctype}`.name"
+	fieldname = f"`tab{args.doctype}`.name"
```

Comment:
This is required when child tables are joined, so you want count of distinct records and not just rows. 

Outer `distinct count(*)` probably makes no sense.